### PR TITLE
tfsec: 0.27.0 -> 0.36.10

### PR DESCRIPTION
**Pull request by anhdle14**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/analysis/tfsec/default.nix [hunk 1]
```diff
@@ -2,23 +2,23 @@
...
 
   buildFlagsArray = [ "-ldflags=-s -w -X ${goPackagePath}/version.Version=${version}" ];
 
   meta = with lib; {
-    homepage = "https://github.com/liamg/tfsec";
+    homepage = "https://github.com/tfsec/tfsec";
     description = "Static analysis powered security scanner for your terraform code";
     license = licenses.mit;
-    maintainers = [ maintainers.marsam ];
+    maintainers = [ maintainers.anhdle14 ];
```
Comment (drewrisinger) [inline pkgs/development/tools/analysis/tfsec/default.nix:22]:
```suggestion
    maintainers = with maintainers; [ marsam anhdle14 ];
```
Why did you remove @marsam? Usually I only remove maintainers with their consent or if the package has been unmaintained for YEARS.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -10152,4 +10152,14 @@
...
+  anhdle14 = {
+    name = "Le Anh Duc";
+    email = "anhdle14@icloud.com";
+    github = "anhdle14";
+    githubId = 9645992;
+    keys = [{
+      longkeyid = "rsa4096/0x0299AFF9ECBB5169";
+      fingerprint = "AA4B 8EC3 F971 D350 482E  4E20 0299 AFF9 ECBB 5169";
+    }];
+  };
```
Comment (drewrisinger) [inline maintainers/maintainer-list.nix:10164]: These changes need to be in a separate commit

**Review by drewrisinger — CHANGES_REQUESTED** [review]
Hi! Thanks for making your first contribution to Nixpkgs!

This PR isn't ready for merging yet, but it can be with a few small changes:
* [ ] Diff: Diff mostly looks good, see comment below about maintainers
* [ ] Commits: since you're adding yourself as a maintainer, this needs separated to a separate commit. Convention in ``nixpkgs`` is one commit per package/file, wherever possible.
* [x] This package builds fine via ``nix-review``, so that part is good to go!

File: pkgs/development/tools/analysis/tfsec/default.nix (hunk 1 above)
Comment (anhdle14) [reply pkgs/development/tools/analysis/tfsec/default.nix:22]: Sorry, I will exclude my name from it right now, was thinking that it is for the latest maintainers

**Comment by drewrisinger** [comment]
@anhdle14 You can & should still have the maintainer commit in this PR if you'd like to be the maintainer, it just needs to be a separate commit (commit & push again)

File: pkgs/development/tools/analysis/tfsec/default.nix (hunk 1 above)
Comment (drewrisinger) [reply pkgs/development/tools/analysis/tfsec/default.nix]: No, you're fine to add your name. Many packages in nixpkgs have multiple maintainers. Maintaining mostly is just a loose commitment to try to keep the package up to date, and be notified for testing purposes via GitHub when someone submits a PR for this package (e.g. ``marsam`` should've been notified for this package originally)

The syntax would be as I showed above.

**Comment by anhdle14** [comment]
@drewrisinger, thank you, @marsam is being requested as well. We can ask whether if he happy to pass it or not. I will create a separate commit if he is ok. Thank you.

**Comment by drewrisinger** [comment]
@anhdle14 can you change the commit title to match nixpkgs convention? Something like the PR title (_tfsec: 0.27.0 -> 0.36.10_) would be great. The information you have there can go on the second line. Also, in nixpkgs we usually call it "bumping" vs "pumping" the version :D

**Comment by anhdle14** [comment]
> @anhdle14 can you change the commit title to match nixpkgs convention? Something like the PR title (_tfsec: 0.27.0 -> 0.36.10_) would be great. The information you have there can go on the second line. Also, in nixpkgs we usually call it "bumping" vs "pumping" the version :D

@drewrisinger Thank you for pointing that out, sorry I did not actually read very carefully the rules yet. This is my first PR for Nixpkgs as well. I fixed it as requested.

**Review by drewrisinger — APPROVED** [review]
Great, thanks for the updates. It looks good to me now!
* [x] Diff LGTM
* [x] Commits LGTM
* [x] Builds via ``nix-review`` on x86_64-linux:
```
https://github.com/NixOS/nixpkgs/pull/106779
1 package built:
tfsec
```

**Review by marsam — APPROVED** [review]
LGTM. Thank you, sorry for the delay
